File: ui-essen/src/app/restaurant.model.ts
export class RestaurantModel {


    restaurantId: number;
    restaurantName: string;
    restaurantLocation: string;

    /**
     * Getter $restaurantLocation
     * @return {string}
     */
    public get $restaurantLocation(): string {
        return this.restaurantLocation;
    }

    /**
     * Setter $restaurantLocation
     * @param {string} value
     */
    public set $restaurantLocation(value: string) {
        this.restaurantLocation = value;
    }

    /**
     * Getter $restaurantId
     * @return {number}
     */
    public get $restaurantId(): number {
        return this.restaurantId;
    }

    /**
     * Getter $restaurantName
     * @return {string}
     */
    public get $restaurantName(): string {
        return this.restaurantName;
    }

    /**
     * Setter $restaurantId
     * @param {number} value
     */
    public set $restaurantId(value: number) {
        this.restaurantId = value;
    }

    /**
     * Setter $restaurantName
     * @param {string} value
     */
    public set $restaurantName(value: string) {
        this.restaurantName = value;
    }

}

File: backend-essen/src/main/java/com/essen/model/RestaurantModel.java
package com.essen.model;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class RestaurantModel {

	private int restaurantId;
	private String restaurantName;
	private String restaurantLocation;
	private String restaurantContactNo;
	private String restaurantAddress;
	private String restaurantCategory;
	private int restaurantExpense;
}
